cms/static/js/certificates/spec/views/certificate_editor_spec.js

```diff
@@ -90,6 +90,16 @@ function(_, Course, CertificateModel, SignatoryModel, CertificatesCollection, Ce
         uploadDialogTpl = readFixtures('upload-dialog.underscore');
 
         beforeEach(function() {
+            window.course = new Course({
```

Comment:
Nit: create a helper function for setting up these global variables so that they don't have to be duplicated into each test.